plugins/out_stackdriver/stackdriver.c

```diff
@@ -644,17 +675,14 @@ static int stackdriver_format(struct flb_config *config,
                             ctx->instance_id, flb_sds_len(ctx->instance_id));
     }
     else if (strcmp(ctx->resource, "k8s_container") == 0) {
-      /* k8s_container resource has fields project_id, location, cluster_name,
-                                          namespace_name, pod_name, container_name */
+        /* k8s_container resource has fields project_id, location, cluster_name,
+                                             namespace_name, pod_name, container_name */
 
         ret = process_local_resource_id(data, bytes, ctx, "k8s_container");
         if (ret != 0) {
-                
-            flb_plg_error(ctx->ins, "Fail to process local_resource_id from log entry");
+            flb_plg_error(ctx->ins, "fail to process local_resource_id from "
+                          "log entry for %s", "k8s_container");
```

Comment:
Is it necessary to use %s here? Why not just use the k8s_container string in the log message

Reply:
removed